binding/react/README.md

```diff
@@ -0,0 +1,227 @@
+# Leopard Binding for React
+
+## Leopard Speech-to-Text Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Leopard is an on-device speech-to-text engine. Leopard is:
+
+- Private; All voice processing runs locally.
+- [Accurate](https://picovoice.ai/docs/benchmark/stt/)
+- [Compact and Computationally-Efficient](https://github.com/Picovoice/speech-to-text-benchmark#rtf)
+- Cross-Platform:
+  - Linux (x86_64), macOS (x86_64, arm64), Windows (x86_64)
+  - Android and iOS
+  - Chrome, Safari, Firefox, and Edge
+  - Raspberry Pi (4, 3) and NVIDIA Jetson Nano
+
+## Compatibility
+
+- Chrome / Edge
+- Firefox
+- Safari
+
+### Restrictions
+
+IndexedDB is required to use `Leopard` in a worker thread. Browsers without IndexedDB support
+(i.e. Firefox Incognito Mode) should use the [`LeopardWeb binding`](https://github.com/Picovoice/leopard/tree/master/binding/web) in the main thread.
```

Comment:
This is `LeopardWeb` description. You should take a look at Porcupine/Rhino's react readme